base/inc/widgets/base-slider.class.php

```diff
@@ -309,19 +309,27 @@ function video_code( $videos, $classes = array() ){
 
 		foreach($videos as $video) {
 			if( empty( $video['file'] ) && empty ( $video['url'] ) ) continue;
-
-			if( empty( $video['url'] ) ) {
-				$video_file = wp_get_attachment_url($video['file']);
-				$video_element .= '<source src="' . sow_esc_url( $video_file ) . '" type="' . esc_attr( $video['format'] ) . '">';
-			}
-			else {
-				$args = '';
```

Comment:
I think we should keep this around for the case where we don't have `$video['height']` defined. It should perhaps rather be initialized to an empty `array()`.